listener/upload.go

```diff
@@ -18,16 +18,25 @@ import (
 func uploadHandler(event PlatformEvent) {
 	if event.B64Identity == nil {
 		utils.Log().Error("Identity not provided")
+		messagesReceivedCnt.WithLabelValues("error-identity").Inc()
```

Comment:
Please add prefix or separate metrics to differentiate between upload & delete messages.

Reply:
ok, done